templates/user/submit/submit.html.twig

```diff
@@ -19,8 +19,10 @@
 {% endblock %}
 
 {% block _submit_users_label %}
-    <label for="{{ full_name }}" class="d-block">Users</label>
-    {{ form_errors(form) }}
+    <label for="{{ full_name }}" class="d-block">
+        Users
+        {{ form_errors(form) }}
+    </label>
```

Comment:
It's weird to display errors inside a `label` tag

Reply:
This is how the bootstrap layout naturally does it, it fits better ike this, or I would have to add some CSS. I found it quite weird too